src/App.js

```diff
@@ -0,0 +1,52 @@
+import ironhackLogo from "./images/ironhack-logo.png";
+import menu from "./images/menu.png";
+import icon1 from "./images/icon1.png";
+import icon2 from "./images/icon2.png";
+import icon3 from "./images/icon3.png";
+import icon4 from "./images/icon4.png";
+import "./App.css";
+
+function App() {
+  return (
+    <div className="App">
+      <div className="main-container">
+        <nav className="nav-container">
+          <img src={ironhackLogo} alt="ironhackLogo" />
+          <img src={menu} alt="menu" className="menu-logo" />
+        </nav>
+        <div className="title-container">
+          <h1>Say hello to ReactJS</h1>
+          <h4>
+            You will learn how to use the most popular frontend library, and
+            become a super Ninja developer.
+          </h4>
+          <button>Awesome!</button>
+        </div>
+      </div>
+      <div className="items-container">
+        <div className="item">
+          <img src={icon1} alt="icon-1" />
```

Comment:
That would increase the usability of your application if the image doesn't load and for users that uses screen readers

```suggestion
          <img src={icon1} alt="Declarative Icon" />
```